bare/config/frontend.jsonnet

```diff
@@ -8,7 +8,7 @@ local common = import 'common.libsonnet';
     authenticationPolicy: { allow: {} },
   }],
   schedulers: {
-    'local': { address: 'localhost:8982' },
+    'local': { endpoint: { address: 'localhost:8982' } },
```

Comment:
I think you will need to change the key of this entry to be `''`. The default behavior is to now strip the part of the instance name that is provided here.

Reply:
fixed.